ballerina-tests/README.md

```diff
@@ -0,0 +1,40 @@
+## Writing Tests
+
+The tests for the Ballerina GraphQL module are organized into multiple test suites. When adding a new test, ensure that it is added to the appropriate test suite. Below are the current test suites:
+
+- **graphql-advanced-test-suite**: Covers test cases for GraphQL context, caching, file uploads, and query complexity.
+- **graphql-client-test-suite**: Covers test cases for the Ballerina GraphQL client.
+- **graphql-dataloader-test-suite**: Covers test cases related to the DataLoader implementation.
+- **graphql-interceptor-test-suite**: Covers test cases for interceptors.
+- **graphql-security-test-suite**: Covers test cases for security.
+- **graphql-subgraph-test-suite**: Covers federation and subgraph-related implementation.
+- **graphql-subscription-test-suite**: Covers subscription-related functionality.
+- **graphql-service-test-suite**: Covers the remaining overall GraphQL specification.
+
+### Test Structure
+
+### Test Structure
+
+- **Listeners**: Test listeners are defined in the `<suite>/tests/listeners.bal` file.
+- **Services**: Test services are defined in the `<suite>/tests/services.bal` file.
+- **Records, Object Types, and Values**: Ballerina records, object types, and values are defined in the corresponding Ballerina files:
+  - `<suite>/tests/records.bal`
+  - `<suite>/tests/object_types.bal`
+  - `<suite>/tests/values.bal`
+
+### Handling Large Documents and JSON Files
```

Comment:
Shall we check whether we need to use sentence-case here?

Reply:
Previous README.md file created for the the tests had Title case, hence following the same here.